### PR TITLE
Bump terraform version 1.7.3

### PR DESCRIPTION
**Pull request by v1gnesh**
Refactor https://github.com/ZOSOpenTools/terraformport/pull/6 to make patches reusable.

### CODE REVIEW

**Comment by v1gnesh** [comment]
/run test again as the prev one stalled / timed out for some reason.

**Comment by v1gnesh** [comment]
/run tests

Prev one stalled / timed out for some reason.

File: buildenv [hunk 1]
```diff
@@ -33,29 +20,63 @@ export ZOPEN_CLEAN="zopen_clean"
...
+  cd .. && echo ""
+  git clone -c advice.detachedHead=false --branch $AFERO_TAG https://github.com/spf13/afero.git
+  echo "Checking out $AFERO_TAG"
+  cd afero
+  # curl -s -o afero--$AFERO_TAG.patch "https://raw.githubusercontent.com/ZOSOpenTools/wharf/main/deps-patches/afero--$AFERO_TAG.patch"
+  # git apply -v --ignore-space-change --ignore-whitespace afero--$AFERO_TAG.patch
+  git apply -v --ignore-space-change --ignore-whitespace ../deps-patches/afero--$AFERO_TAG.patch
+
+  # cd .. && echo ""
+  # git clone https://github.com/bgentry/speakeasy.git
```
Comment (cpkr162) [inline buildenv:63]: @v1gnesh I think, we can remove speakeasy clone since it's handled in go.mod.patch file

Comment (v1gnesh) [reply buildenv]: Yup, removed the comments now

File: buildenv [hunk 2]
```diff
@@ -1,30 +1,17 @@
...
+TERRAFORM_VERSION="1.7.3"
 
 export ZOPEN_BUILD_LINE="STABLE"
 
 export ZOPEN_STABLE_TAG="v${TERRAFORM_VERSION}"
 export ZOPEN_STABLE_URL="https://github.com/hashicorp/terraform.git"
-export ZOPEN_STABLE_DEPS="comp_go wharf"
-
+export ZOPEN_STABLE_DEPS="comp_go git curl"
 export ZOPEN_CATEGORIES="development"
```
Comment (cpkr162) [inline buildenv:9]: May be, we can add devops to ZOPEN_CATEGORIES

**Comment by v1gnesh** [comment]
Ready to merge

**Pull request merged**